config.go

```diff
@@ -110,6 +130,25 @@ type config struct {
 	ProxyUser        string `long:"proxyuser" description:"Username for proxy server"`
 	ProxyPass        string `long:"proxypass" default-mask:"-" description:"Password for proxy server"`
 
+	// Automatic ticket buyer settings
```

Comment:
OK, removed. Passing defaults for all the configuration options instead. Config can be updated later, with provided specific parameters, if necessary.